README.md

```diff
@@ -65,11 +65,48 @@ TODO_BOARD_ID
 ### gunicorn support
 Support for gunicorn local run 
 ```
-poetry run gunicorn --daemon --bind 0.0.0.0:5000 -w 1 "wsgi:create_app()"
+$ poetry run gunicorn --daemon --bind 0.0.0.0:5000 -w 1 "wsgi:create_app()"
 ```
 
 ### Vagrant
 You can also use vagrant
 ```
 $ vagrant up
+```
+
+### Docker Compose
+
+```
+$ docker-compose build && docker-compose up -d <target>
+```
+
+
+### Docker
+
+Build the container
+```
+$ docker build --tag todo-app .
```

Comment:
If you don't specify a target, this will build the last stage which is not what you want here.